pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -301,6 +302,11 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 			err = errors.Wrapf(err, "could not apply (%s) %s/%s", obj.GroupVersionKind(), obj.GetNamespace(), obj.GetName())
 			log.Println(err)
 
+			// If error comes from nonexistent namespace print out a help message.
+			if obj.GroupVersionKind().Kind == "NetworkAttachmentDefinition" && strings.Contains(err.Error(), "namespaces") {
+				log.Println("Namespace error for networkattachment definition, consider possible solutions: (1) Edit config files to include existing namespace (2) Create non-existent namespace (3) Delete erroneous network-attachment-definition")
```

Comment:
Just logging this to stdout isn't likely to get noticed. The suggestion in bz1896533 was that if CNO is going to become Degraded because of this, then it needs to explain why _in the Degraded status message_. (In this context, that means you want to change the value of `err`, so as to change what gets passed to `r.status.SetDegraded` below.)

But I also think that the reporters are correct and there are good reasons to not consider this case to be an error, so I'm not sure this patch is right anyway.